homework03/go.mod

```diff
@@ -0,0 +1,3 @@
+module Golang/epam/goCourse
```

Comment:
If you're using go mod inside of a single folder, it should lead to this folder... In this case it should be:
`module Golang/epam/goCourse/homework03`